npm audit: add page (#14499)
tldr-pages/tldr: pages/common/npm-audit.md

# npm audit

> > Command to scan for known vulnerabilities in project dependencies.

Reviewer: ```suggestion
> Scan for known vulnerabilities in project dependencies.
```

> Reports vulnerabilities and suggests remediation.
> More information: <https://docs.npmjs.com/cli/npm-audit>.

> - Scan project’s dependencies for known vulnerabilities:

Reviewer 2: ```suggestion
- Scan the project’s dependencies for known vulnerabilities:
```

`npm audit`

> - Automatically fix vulnerabilities:

Reviewer 2: ```suggestion
- Automatically fix vulnerabilities in the project's dependencies:
```

`npm audit fix`

> 

Reviewer 2: ```suggestion

- Force an automatic fix to dependencies with vulnerabilities:

`npm audit fix {{-f|--force}}`

```

> - Check your project for vulnerabilities-  Scans your project’s dependencies for known vulnerabilities:
> 
> `npm audit`
> 
> - Automatically fix vulnerabilities-  Attempts to fix vulnerabilities by updating to safe package versions:
> 
> `npm audit fix`
> 
> - Update only the `package-lock.json`-  Updates the lock file without modifying the `node_modules` directory:
> 
> `npm audit fix --package-lock-only`
> 
> - Perform a dry run-  Simulates the fix process without making any changes:
> 
> `npm audit fix --dry-run`
> 
> - Output audit results in JSON format-  Provides audit results in a structured JSON format for processing:
> 
> `npm audit --json`
> 
> - Set a minimum severity level- Configures the audit to only fail on vulnerabilities above a specified severity:

Reviewer: ```suggestion
- Scan project’s dependencies for known vulnerabilities:

`npm audit`

- Automatically fix vulnerabilities:

`npm audit fix`

- Update the lock file without modifying the `node_modules` directory:

`npm audit fix --package-lock-only`

- Perform a dry run. Simulate the fix process without making any changes:

`npm audit fix --dry-run`

- Output audit results in JSON format:

`npm audit --json`

- Configure the audit to only fail on vulnerabilities above a specified severity:
```

Author: got it this will help me push better docs
thanks! @spageektti
